src/main/java/com/google/gwtorm/server/StandardKeyEncoder.java

```diff
@@ -0,0 +1,114 @@
+// Copyright 2008 Google Inc.
```

Comment:
Please exclude this file from test coverage calculation; see: https://github.com/uwolfer/gerrit-rest-java-client/blob/master/pom.xml#L128